### PR TITLE
Include fewer paths in exception

### PR DESCRIPTION
**Pull request by amcasey**
It's sufficient to add them to the log.

### CODE REVIEW

**Comment by sheetalkamat** [comment]
you need to update the failing test case

File: src/server/session.ts [hunk 1]
```diff
@@ -711,7 +711,9 @@ namespace ts.server {
                     catch { } // tslint:disable-line no-empty
                 }
 
-                if (err.message && err.message.indexOf(`Could not find sourceFile:`) !== -1) {
+
+                if (err.ProgramFiles) {
+                    msg += `\n\nProgram files: {JSON.stringify(error.ProgramFiles}}\n`;
```
Comment (sheetalkamat) [inline src/server/session.ts]: This needs to be 
 msg += `\n\nProgram files: ${JSON.stringify(error.ProgramFiles)}\n`;

Comment (amcasey) [reply src/server/session.ts]: That's what I get for inspecting the property in the debugger and not actually reading the log...

Comment (amcasey) [reply src/server/session.ts]: #32789

**Pull request merged**